README.md

```diff
@@ -154,7 +180,7 @@ How many valid sentences are there in this language?
 
 ## A2.6
 
-«replace this with your answer»
+((2*5)*2)*3 -> 60
```

Comment:
# -2

In a version with no repeated adjectives that might be right. But the actual language has an infinite number of sentences.